src/screens/EnquiryManagement.js

```diff
@@ -0,0 +1,226 @@
+import React, { useState } from "react";
+import { SafeAreaView, Text, TextInput, TouchableOpacity, View, Alert, StyleSheet } from 'react-native';
+import firestore from '@react-native-firebase/firestore';
+
+const Enquiry = ({ navigation }) => {
+    const go = () => {
+        navigation.navigate('Query');
+    }
+    const go1 = () => {
+        navigation.navigate('Feedback');
+    }
+
+    return (
+        <SafeAreaView style={styles.container}>
+            <View>
+                <TouchableOpacity onPress={go} style={{
+                    backgroundColor: '#E7B909',
+                    borderRadius: 8,
+                    padding: 16,
+                    marginBottom: 16,
+                    shadowColor: '#000',
+                    shadowOffset: { width: 0, height: 2 },
+                    shadowOpacity: 0.2,
+                    shadowRadius: 4,
+                    elevation: 2,
+                    marginHorizontal: 30,
+                    marginVertical: 300,
+                    width: '80%',
+                }}>
+                    <Text style={{
+                        color: '#B02A30',
+                        fontWeight: 'bold',
+                        fontSize: 20,
+                        textAlign: 'center',
+                        borderColor: 'black'
+                    }}>Post a Query</Text>
+                </TouchableOpacity>
+                <TouchableOpacity onPress={go1} style={{
```

Comment:
Give logical / relevant name. "go1" doesn't make any sense.
for eg. "navigateToQuery"